src/ratings.html

```diff
@@ -58,14 +63,5 @@
 		<p id="|content">
 		</p>
 		<script src="js/ratinglist.js"></script>
-		<!-- Google tag (gtag.js) -->
```

Comment:
since we're getting rid of google analytics here I'm wondering if we can at least add our new relic script to get some observability on this page

Reply:
What do you want to observe? I think it's already so heavy with 70k elements, any analytics is going to be laggy since it will probably be adding listeners to every element.

Reply:
new relic does a few things such as performance monitoring, error logging, and region performance tracking. I don't I've seen a negative impact on performance using new relic.

Reply:
Well, feel free to add it. We can always analyze performance after and see whether it's having an impact.